tests/fixtures/lxd-test.ts

```diff
@@ -0,0 +1,10 @@
+import { test as base } from "@playwright/test";
+
+export type LxdVersions = "5.0-stable" | "latest-edge";
+export type TestOptions = {
+  lxdVersion: LxdVersions;
+};
+
+export const test = base.extend<TestOptions>({
+  lxdVersion: ["latest-edge", { option: true }],
```

Comment:
I am confused as the value passed here seems to be hardcoded. Is the actual value coming from the use block in playwright config? Maybe we don't need the value here, then?

Though, please keep this lxd-test file, because I am going to need it for the [coverage](https://github.com/canonical/lxd-ui/pull/634) very soon -- even if it turns out to be empty and just passes the base test at this point.

Reply:
The value set here is the default for the project `use` parameter. I referenced the playwright docs for [parameterising projects](https://playwright.dev/docs/test-parameterize#parameterized-projects). I think { option: true } is necessary but not sure if we can pass in a empty value for the first parameter of the array. Although passing in an empty value will look strange as well I rate, wdyt?

Reply:
If we need this definition here, I think keeping `latest-edge` as a default is reasonable.

Maybe we should document the test setup, as it lately became a bit more complex. Maybe a future story to extend the [architecture file](https://github.com/canonical/lxd-ui/blob/main/ARCHITECTURE.MD)?

Reply:
will do, I did add a bit of context in the HACKING.md, but I think adding to the architecture file is also necessary. Will create a task for it :+1: